source/MaterialXTest/GenOgsXml.cpp

```diff
@@ -48,6 +48,13 @@ class OgsXmlShaderGeneratorTester : public GlslShaderGeneratorTester
         _testStages.push_back(mx::Stage::PIXEL);
     }
 
+    // Ignore light shaders in the document for GLSL fragments since MaterialX
+    // lights and not supported in OGS/VP2.
+    void findLights(mx::DocumentPtr /*doc*/, std::vector<mx::NodePtr>& lights) override
```

Comment:
The base class registers lights found in the document which produces some really strange results during GLSL generation. MaterialX lights are not supported in the Maya integration however.